src/app/app.module.ts

```diff
@@ -23,10 +29,12 @@ import { webpackLoader } from './loaders/webpack.loader';
       useValue: {
         prodMode: environment.production,
         listenToLangChange: true,
-        fallbackLang: 'es',
+        fallbackLang: 'en',
```

Comment:
why changed? 🤔

Reply:
My mistake. It's leftovers from debugging. Will remove.